lib/src/openApiToZod.ts

```diff
@@ -146,6 +148,15 @@ export function getZodSchema({ schema, ctx, meta: inheritedMeta, options }: Conv
                 return code.assign("z.never()");
             }
 
+            if (schema["x-enumNames"]?.length === schema.enum?.length) {
```

Comment:
also, do you think it would make sense here to include a check on `length === 1` like the others to simplify the output when possible ?

Reply:
Don't believe so. As a consumer of the library I'd expect a specification stating "there is an enum Foo with value Bar = 42" to allow the following code `Foo.Bar == 42`, that the enum happens to be a singleton doesn't really matter.

The non-labeled enum case is fine as it effectively doesn't change the exposed interface.